docker/linux/Dockerfile

```diff
@@ -24,4 +24,4 @@ RUN pip --no-cache-dir install -U iotedgedev
 WORKDIR /home/iotedge
 COPY install-dev.sh /scripts/install-dev.sh
 RUN sed -i 's/\r//' /scripts/install-dev.sh
-ENV DEBIAN_FRONTEND teletype
\ No newline at end of file
+ENV DEBIAN_FRONTEND teletype
```

Comment:
What is the change here?  If no changes, then please remove from the commit.  Thanks,

Reply:
Yep